File: 2574 left-and-right-sum-differences/solution.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int lsum=0,rsum=0;
        for(int i:nums) rsum+=i;
        for(int i=0;i<nums.length;i++) {
            int l=nums[i];
            rsum-=nums[i];
            nums[i]=Math.abs(lsum-rsum);
            lsum+=l;
        }
        return nums;
    }
}

File: 2574 left-and-right-sum-differences/solution.cs
public class Solution {
    public int[] LeftRightDifference(int[] nums) {
        int n=nums.Length;
        int lsum=0,rsum=0;
        foreach(int i in nums) lsum+=i;
        for(int i=0;i<n;i++) {
            int l=nums[i];
            rsum+=l;
            nums[i]=Math.Abs(lsum-rsum);
            lsum-=l;
        }
        return nums;
    }
}
